pkg/controller/storage/mem/storage.go

```diff
@@ -24,17 +24,17 @@ import (
 type memStorage struct {
 	brokers        *memStorageBroker
 	serviceClasses *memStorageServiceClass
-	instances      map[string]*memStorageInstance
-	bindings       map[string]*memStorageBinding
+	instances      map[string]storage.InstanceStorage
+	bindings       map[string]storage.BindingStorage
```

Comment:
@duglin there's hardly ever a reason to pass a pointer to an interface. I've never seen one in practice, and I can't think of a reason to do so off the top of my head. Hopefully that helps?